src/documentation/markdown/2019/static-dataset-faq.md

```diff
@@ -1,30 +1,30 @@
-## Static Dataset FAQ
+# Static Dataset FAQ
 
-### What is the difference between the static datasets?
+## What is the difference between the static datasets?
 The primary difference between the datasets is the time period when the data is captured. Each time period contains the most up-to-date data _at the moment in time when the dataset was created_.
 
 |Dataset|Time Period|
 |---|---|
-|Snapshot | 1 month after the timely filing deadline |
+|Snapshot | Typically 1 month |
```

Comment:
```suggestion
|Snapshot | Typically 1 month after the timely filing deadline|
```

Reply:
Sounds good. I have reverted the change to the original. I agree Time of Capture sounds better, I have updated the header for the table.

Changes are included from 2017-2023.